### ISSUE TITLE
kubectl describe nodes id should include namepaces when dumping pods running on the node

### ISSUE THREAD

**Issue by dchen1107** [GoogleCloudPlatform/kubernetes#9433]
kubectl describe nodes id today shows all pods running on a node with that id, but doesn't include namespace for each pod:
```
$ cluster/kubectl.sh describe pods dns-test-fbcde65a-0e23-11e5-aa37-24be050deb0e
dawnchen@dawndev3:/usr/local/src/goproj/src/github.com/GoogleCloudPlatform/kubernetes$ 
dawnchen@dawndev3:/usr/local/src/goproj/src/github.com/GoogleCloudPlatform/kubernetes$ cluster/kubectl.sh describe nodes e2e-test-dawnchen-minion-r27t
W0608 14:19:21.456425   16588 request.go:291] field selector: v1beta3 - events - involvedObject.namespace - : need to check if this is versioned correctly.
W0608 14:19:21.456506   16588 request.go:291] field selector: v1beta3 - events - involvedObject.kind - Node: need to check if this is versioned correctly.
W0608 14:19:21.456518   16588 request.go:291] field selector: v1beta3 - events - involvedObject.uid - e2e-test-dawnchen-minion-r27t: need to check if this is versioned correctly.
W0608 14:19:21.456529   16588 request.go:291] field selector: v1beta3 - events - involvedObject.name - e2e-test-dawnchen-minion-r27t: need to check if this is versioned correctly.
Name:			e2e-test-dawnchen-minion-r27t
Labels:			kubernetes.io/hostname=e2e-test-dawnchen-minion-r27t
CreationTimestamp:	Mon, 08 Jun 2015 13:18:21 -0700
Conditions:
  Type		Status	LastHeartbeatTime			LastTransitionTime			Reason					Message
  Ready 	True 	Mon, 08 Jun 2015 14:19:15 -0700 	Mon, 08 Jun 2015 14:12:34 -0700 	kubelet is posting ready status 	
Addresses:	104.197.27.218,104.197.27.218
Capacity:
 cpu:		1
 memory:	1745136Ki
 pods:		100
Version:
 Kernel Version:		3.16.0-0.bpo.4-amd64
 OS Image:			Debian GNU/Linux 7 (wheezy)
 Container Runtime Version:	docker://1.7.0-rc2
 Kubelet Version:		v0.18.1-470-g636b40ffa19928-dirty
 Kube-Proxy Version:		v0.18.1-470-g636b40ffa19928-dirty
PodCIDR:			10.245.1.0/24
ExternalID:			7487646186902049538
Pods:				(2 in total)
  fluentd-elasticsearch-e2e-test-dawnchen-minion-r27t
  dns-test-03e93b16-0e24-11e5-aa37-24be050deb0e
```

But when the user describe pod, it requires the namespace. There is no good way to find such pod belonging to which namespace so far.

**Comment by dchen1107** [comment]
cc/ @bgrant0607

**Comment by dchen1107** [comment]
Some related discussion can be found at #9356

**Comment by hurf** [comment]
I happened to be reading the code of this part recently, I'll create a PR to show namespaces.

**Comment by hurf** [comment]
@dchen1107 - Should we display namespaces of other resources like service, pod, etc. while describing them?

**Comment by brendandburns** [comment]
@hurf yes, that would be great.  Please feel free to send the PR to me.

Thanks!
--brendan

**Comment by brendandburns** [comment]
Closed by #9650
